### ISSUE TITLE
External links in the Steam client won't open

### ISSUE THREAD

**Issue by NoXPhasma** [ValveSoftware/steam-for-linux#4536]
* Steam client version: 1467840262
* Distribution: Linux Mint 18 x64 
* Opted into Steam client beta?: Yes
* Have you checked for system updates?: Yes

When I click on any external Link in the Steam client (Forums, Chat etc.) nothing happens. I get this error on the console when I click on a link:
```
ExecSteamURL: "steam://openurl/https://www.google.com"
sh: 1: STEAM_RUNTIME=0 LD_LIBRARY_PATH="$SYSTEM_LD_LIBRARY_PATH" PATH="$SYSTEM_PATH" /usr/bin/xdg-open: not found
```
However, xdg-utils are installed and xdg-open is present:
```
$ which xdg-open
/usr/bin/xdg-open
```

**Comment by Tele42** [comment]
Related to #4528.

**Comment by predr** [comment]
With latest updates this bug is now in standard client on Ubuntu 16.04.

**Comment by DougTy** [comment]
Also getting this on Arch

**Comment by veluria** [comment]
Please release a fix for this soon. This is one of the most annoying bugs ever.

**Comment by wolrah** [comment]
Confirming the same behavior on a fresh install of Ubuntu 16.04 x64 using the non-beta client.  The xdg-open binary is in /usr/bin/ as expected, but Steam can't find it for whatever reason.

**Comment by NoXPhasma** [comment]
This is fixed in the latest Steam beta update (1468520696):

> Linux
Fixed handling of clickable URLs in chat text for the desktop Steam client

I will close this issue.
